inc/datahandlers/user.php

```diff
@@ -1858,7 +1858,7 @@ public function verify_signature()
 		$parsed_sig = preg_replace("#\s#", "", $parsed_sig);
 		$sig_length = my_strlen($parsed_sig);
```

Comment:
This should be set only if `$mybb->settings['siglength'] > 0` is true. I will accept anyways in case I'm the only one.

Reply:
Good idea, others?